app/scripts/api.js

```diff
@@ -1,14 +1,63 @@
-import { API_URL } from "../constants";
-import { storageSet } from "../store";
-
-export const fetchAndCacheScores = url => {
-  return new Promise((resolve, reject) => {
-    const api = `${API_URL}?url=${url}`;
-    fetch(api)
-      .then(response => response.json())
-      .then(json => {
-        storageSet(url, json).then(() => resolve(json));
-      })
-      .catch(err => reject(err));
-  });
+import axios from "axios";
+import { API_URL, TOKEN_TTL } from "../constants";
+import { storageSet, storageGet } from "../store";
+import { getTokenDuration } from "./utils";
+
+export const validateToken = token => {
+  const duration = getTokenDuration(token.created_at, token.expiry);
+  let isValid = duration <= TOKEN_TTL;
```

Comment:
Do we have a convention that we need to follow to represent variables. Like either `snake_case` or `camelCase`.?

Reply:
@maruthi-adithya anything from the api should be snake case. anything from clientside would be camelcase.